CONTRIBUTORS.md

```diff
@@ -32,7 +32,6 @@ If you would like to contribute to the HELICS project see [CONTRIBUTING](CONTRIB
 - [Ai Enkoji](https://github.com/aenkoji1)
 - [Nan Duan](https://github.com/nan-duan)
 - [Nathan Yee](https://github.com/yee29) (Now Accenture)
-- [Ai Enkoji](https://github.com/aenkoji1)
```

Comment:
It looks like there are a few commits from Ai in the main HELICS repo.

Reply:
I know there was a duplicate line, so she is still there, but no reason to include twice, see line 32

Reply:
Ah right, I missed the lines above Nan in the review tab.